scripts/documentation/generate-cli-data.ts

```diff
@@ -43,6 +46,11 @@ export async function generateCLIDocumentation() {
    */
   process.env.NX_GENERATE_DOCS_PROCESS = 'true';
 
+  const config = readJsonSync(
+    join(__dirname, '../../tsconfig.base.json')
+  ).compilerOptions;
+  registerTsConfigPaths(config);
+
```

Comment:
Talked a bit with @leosvelperez about the changes to this file. Essentially running the import to @nrwl/workspace in this file was causing `yarn documentation` to throw when I had added a new file to @nrwl/tao. From what I can tell this is caused by the imports trying to resolve the files in node_modules instead of obeying paths.

@leosvelperez 's suggestion was a combination of using the `tsconfig-paths` package and updating the import below to not import the whole workspace path. 

These changes are not related to the rest of the file, nor the purpose of the PR, but block this PR so I am including them here.